source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -308,39 +308,39 @@ def parse_and_emit_records(podInventory, serviceRecords, continuationToken, batc
           end
           # Setting this flag to true so that we can send ContainerInventory records for containers
           # on windows nodes and parse environment variables for these containers
-          nodeName = ""
-          if !item["spec"]["nodeName"].nil?
-            nodeName = item["spec"]["nodeName"]
-          end
-          if (!item["isWindows"].nil? && !item["isWindows"].empty? && item["isWindows"].downcase == "true")
-            clusterCollectEnvironmentVar = ENV["AZMON_CLUSTER_COLLECT_ENV_VAR"]
-            #Generate ContainerInventory records for windows nodes so that we can get image and image tag in property panel
-            containerInventoryRecords = KubernetesContainerInventory.getContainerInventoryRecords(item, batchTime, clusterCollectEnvironmentVar, true)
-            if KubernetesApiClient.isEmitCacheTelemetry()
-              @windowsContainerRecordsCacheSizeBytes += containerInventoryRecords.to_s.length
-            end
-            # Send container inventory records for containers on windows nodes
-            @winContainerCount += containerInventoryRecords.length
-            containerInventoryRecords.each do |cirecord|
-              if !cirecord.nil?
-                containerInventoryStream.add(emitTime, cirecord) if cirecord
-                ciRecordSize = cirecord.to_s.length
-                @winContainerInventoryTotalSizeBytes += ciRecordSize
-                if ciRecordSize >= Constants::MAX_RECORD_OR_FIELD_SIZE_FOR_TELEMETRY
-                  @winContainerCountWithInventoryRecordSize64KBOrMore += 1
-                end
-                if !cirecord["EnvironmentVar"].nil? && !cirecord["EnvironmentVar"].empty? && cirecord["EnvironmentVar"].length >= Constants::MAX_RECORD_OR_FIELD_SIZE_FOR_TELEMETRY
-                  @winContainerCountWithEnvVarSize64KBOrMore += 1
-                end
-                if !cirecord["Ports"].nil? && !cirecord["Ports"].empty? && cirecord["Ports"].length >= Constants::MAX_RECORD_OR_FIELD_SIZE_FOR_TELEMETRY
-                  @winContainerCountWithPortsSize64KBOrMore += 1
-                end
-                if !cirecord["Command"].nil? && !cirecord["Command"].empty? && cirecord["Command"].length >= Constants::MAX_RECORD_OR_FIELD_SIZE_FOR_TELEMETRY
-                  @winContainerCountWithCommandSize64KBOrMore += 1
-                end
-              end
-            end
-          end
+          # nodeName = ""
```

Comment:
This shouldnt be commented. Rather this shouldn't be invoked in case of MSI auth.